src/WebAppDIRAC/WebApp/handler/RootHandler.py

```diff
@@ -151,10 +151,10 @@ def web_loginComplete(self, code, state, **kwargs):
                 t.generate(next=authSession["next"], access_token="", message=result["Message"]).decode()
             )
         nextURL = f"/{Conf.rootURL().strip('/')}"
-        if group := result["Value"].get("group"):
-            nextURL += f"/g:{group}"
         if setup := self.getUserSetup():
             nextURL += f"/s:{setup}"
+        if group := result["Value"].get("group"):
+            nextURL += f"/g:{group}"
```

Comment:
@TaykYoku Why did you change this ordering in https://github.com/DIRACGrid/WebAppDIRAC/pull/654? The routing only understands if the setup comes first:

https://github.com/DIRACGrid/WebAppDIRAC/blob/d70c268ff47fc051ca29fd0415fcaeb0e7587f2a/src/WebAppDIRAC/Core/HandlerMgr.py#L37-L40

Reply:
Yes I know. I have already submitted a PR with all the corrections. https://github.com/DIRACGrid/WebAppDIRAC/pull/656

Reply:
Note: It still makes no mention of what the PR does beyond the delighful "fix bugs".

Reply:
Yeap sorry, I forgot to add a list of fixes to the release notes description, now fixed.